docs/content/en/docs/examples/modsecurity.md

```diff
@@ -26,31 +26,33 @@ adjust the steps to fit your need.
 
 The ModSecurity agent used is [jcmoraisjr/modsecurity-spoa](https://github.com/jcmoraisjr/modsecurity-spoa).
 
-Create the ModSecurity agent daemonset:
+Create the ModSecurity agent deployment with 3 running pods:
 
 ```
-$ kubectl create -f https://haproxy-ingress.github.io/resources/modsecurity-daemonset.yaml
-daemonset "modsecurity-spoa" created
+$ kubectl create -f https://haproxy-ingress.github.io/resources/modsecurity-deployment.yaml
+deployment.apps/modsecurity-spoa created
 ```
 
-Select the node(s) where ModSecurity agent should run:
 
-```
-$ kubectl get node
-NAME             STATUS    AGE       VERSION
-192.168.100.99   Ready     102d      v1.9.2
-...
+Check if the agent is up and running:
 
-$ kubectl label node 192.168.100.99 waf=modsec
-node "192.168.100.99" labeled
+```
+$ kubectl -n ingress-controller get deployment modsecurity-spoa
+NAME                     READY     UP-TO-DATE   AVAILABLE  AGE  
+modsecurity-spoa         3/3       3            3          7s   
 ```
 
-Check if the agent is up and running:
 
+You can now create the service that provides a ClusterIP address for the HAProxy ConfigMap.
 ```
-$ kubectl -n ingress-controller get pod -lrun=modsecurity-spoa -owide
-NAME                     READY     STATUS    RESTARTS   AGE       IP               NODE
-modsecurity-spoa-pp6jz   1/1       Running   0          7s        192.168.100.99   192.168.100.99
+$ kubectl -n ingress-controller expose deployment modsecurity-spoa --port=12345 --type=ClusterIP
+```
+
```

Comment:
Missing a `service/modsecurity-spoa exposed` below this expose, just to follow the convention of the doc.